hazelcast-client-new/src/test/java/com/hazelcast/client/executor/ClientExecutorServiceTest.java

```diff
@@ -206,7 +206,7 @@ public void onFailure(Throwable t) {
     }
 
 
-    @Test(expected = IllegalStateException.class)
+    @Test
     public void testSubmitFailingCallableReasonExceptionCause() throws Throwable {
```

Comment:
This is another feature that we lost via my implementation. One solution is to add cause error code definition of exception in protocol. Question there is how many levels of cause we should put? For this test one more level is enough. But generally every exception may have a cause.